src/ontology/cl-edit.owl

```diff
@@ -27032,6 +27034,27 @@ EquivalentClasses(obo:CL_3000004 ObjectIntersectionOf(obo:CL_0000111 ObjectSomeV
 SubClassOf(obo:CL_3000004 obo:CL_0000101)
 SubClassOf(obo:CL_3000004 obo:CL_0000111)
 
+# Class: obo:CL_4023000 (beta motor neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25346659") obo:IAO_0000115 obo:CL_4023000 "A motor neuron that innervates both intrafusal and extrafusal muscle fibers. Not well characterised due to its low abundancy. They control both muscle contraction and responsiveness of the sensory feedback from muscle spindles.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023000 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023000 "beta motor neuron")
+SubClassOf(obo:CL_4023000 obo:CL_0008039)
+
+# Class: obo:CL_4023001 (static beta motor neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25346659") obo:IAO_0000115 obo:CL_4023001 "A beta motor neuron that innervates nuclear chain fibers.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023001 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023001 "static beta motor neuron")
+SubClassOf(obo:CL_4023001 obo:CL_4023000)
+
+# Class: obo:CL_4023002 (dynamic beta motor neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25346659") obo:IAO_0000115 obo:CL_4023002 "A beta motor neuron that innervates the nuclear bag fibers of muscle spindles.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023002 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023002 "dynamic beta motor neuron")
+SubClassOf(obo:CL_4023002 obo:CL_4023000)
+
```

Comment:
Please make tickets for spindle component 'nuclear chain fiber' and 'nuclear bag fiber'